apps/web/src/services/asset-inventory/collector/modules/CollectorContents.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+    <div class="collector-contents">
+        <collector-provider-list />
+        <p-toolbox
+            search-type="query"
+            :total-count="props.totalCount"
+            :page-size="props.pageLimit"
```

Comment:
Question: Is `pageLimit` reflected when requesting api? 🤔

Reply:
When using the toolbox, I used pageLimit to display the total size of pages,,!